### DISCUSSION TITLE
Running SO in a passive / active datacenter

### DISCUSSION THREAD

**Discussion by michielmeersmans** [Security-Onion-Solutions/securityonion#3780] [Unsupported Versions]
I'm looking in deploying security onion in a setup with a passive / active datacenter.

Under normal circumstances, the passive DC does not receive any traffic. I'd like to have the events from the active DC synced to the passive so when an error and failover switch happens, the events are stored there too.
One option would be to use cross cluster replication in xpack (https://www.elastic.co/guide/en/elasticsearch/reference/current/xpack-ccr.html), but I'm not sure if I would still need to mirror the master node in a way.

I was also looking at solutions that do not require x-pack licenses. For example by changing the logstash config on the master in the active DC to also forward logs to the logstash instance of the passive DC.

Does anyone have experience with how to tackle this?

**Comment by Acewiza** [comment]
That's going to be quite a challenge considering some people seem to have issues keeping just one standalone operating correctly.
